packages/studioCMS/src/components/exports/navigation.css

```diff
@@ -0,0 +1,148 @@
+.navigation {
```

Comment:
Haven't checked, but is this element a `position: relative` or other so that the `.title` is positioned absolutely relatively to this parent

Reply:
Yeah its a big Flex